README.rst

```diff
@@ -18,6 +18,28 @@ dealing with the Halo Model of Dark Matter Halos, built on
 
 It also has a full-feature web application at https://thehalomod.app -- check it out!
 
+HI Branch
+---------
+This branch introduces new model based on Chen, Wolz, Spinelli and Murray.
+For the HI model used and how to make your own extension, see https://halomod.readthedocs.io/en/latest/examples/extension.html .
+
+To use our model, simply:
+
+>>> from halomod import TracerHaloModel
+>>> hm=TracerHaloModel(hod_model="Spinelli19",
+>>>                    tracer_concentration_model="Maccio07",
+>>>                    tracer_profile_model="PowerLawWithExpCut",
+>>>                    z=1 #for default value of hod parameters
+>>>                   )
+
+
+To calculate the HI power spectrum:
+
+>>> ps_hi = hm.power_auto_tracer
+
+To calculate the temperature power spectrum:
+
+>>> ps_t = hm.power_auto_tracer*hm.mean_tracer_den_unit**2
```

Comment:
I would leave this to an added notebook, rather than the README. We can point to the notebook directly from the paper as well.

Reply:
Yeah it was just like a temporary thing for the HI branch and then i forgot about it. I would just add it to the existing notebook which is about that anyway.

> Also, is the version at the head of this branch currently exactly the one you used in the paper? I could create a tag that points here for posterity. It might be nicer though if you could literally just say you used "v2.0.0" of halomod -- which I hope will be released in the next week or two.

Not exactly, the actual version used in the paper is slightly different (basically the one last december or january? but the extensions are the same), but I think it has no fundamental difference and it should work just fine anyway.

Reply:
Yep, looks like I haven't updated the GitHub Actions scripts for a while. I'll be putting up a PR today to be merged -- that can then be merged into this, and will fix this issue.